peps/pep-0738.rst

```diff
@@ -0,0 +1,97 @@
+PEP: 738
+Title: Enabling certificate verification by default for stdlib mail modules
+Author: Martin Schobert <martin@pentagrid.ch>
+Status: Draft
+Type: Standards Track
+Created: 17-Nov-2023
+
+Abstract
+=========
+
+The Python mail libraries such as ``smtplib``, ``imaplib`` and ``pop3lib``
+do not verify server certificates per default, when a client based on these
+mail libraries connects to a server via TLS. Any server certificate is
+accepted per default. This means, a client can't ensure that it connects
+to the server to which the connection was intended.
+
+This allows an active attacker in a machine-in-the-middle postion to intercept
+communication, read mail contents, credentials or may abuse an SMTP server
+for spaming.
+
+Motivation
+===========
+
+This PEP proposes to enable verification of X509 certificates for Python's
+mail clients by default, subject to opt-out on a per-call basis. This change
+would be applied to all maintained Python versions.
```

Comment:
This sounds like the abstract! And what you wrote in the abstract sounds like the motivation. :-)